src/App.tsx

```diff
@@ -6,7 +6,7 @@ import { GlobalStyles } from './styles/global'
 import { BrowserRouter as Router } from 'react-router-dom'
 
 function App() {
-  const [theme, setTheme] = useState('light')
+  const [theme, setTheme] = useState<'light' | 'dark'>('light')
```

Comment:
use enum

something like this
```
enum Theme {
  light = 'light',
  dark = 'dark',
}

```

and use   const [theme, setTheme] = useState<Theme>(Theme.light)